internal/provider/organizations_claim_resource_test.go

```diff
@@ -0,0 +1,123 @@
+				),
+			},
+
+			// Create and Read OrganizationsClaim
+			{
+				Config: testAccOrganizationsClaimResourceConfigCreate(os.Getenv("TF_ACC_MERAKI_ORDER_NUMBER"), os.Getenv("TF_ACC_MERAKI_MX_SERIAL"), os.Getenv("TF_ACC_MERAKI_MX_LICENCE")),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("OrganizationsClaim.test", "id", "example-id"),
+
+					resource.TestCheckResourceAttr("organizations_claim.test", "orders.#", "1"),
+					resource.TestCheckResourceAttr("organizations_claim.test", "orders.0", os.Getenv("TF_ACC_MERAKI_ORDER_NUMBER")),
+
+					resource.TestCheckResourceAttr("organizations_claim.test", "serials.#", "1"),
+					resource.TestCheckResourceAttr("organizations_claim.test", "serials.0", os.Getenv("TF_ACC_MERAKI_MX_SERIAL")),
+
+					resource.TestCheckResourceAttr("organizations_claim.test", "licenses.#", "1"),
+					resource.TestCheckResourceAttr("organizations_claim.test", "licenses.0.key", os.Getenv("TF_ACC_MERAKI_MX_LICENCE")),
+					resource.TestCheckResourceAttr("organizations_claim.test", "licenses.0.mode", "addDevices"),
+				),
+			},
+
+			// Update and Read OrganizationsClaim
+			{
+				Config: testAccOrganizationsClaimResourceConfigUpdate(os.Getenv("TF_ACC_MERAKI_ORDER_NUMBER"), os.Getenv("TF_ACC_MERAKI_MX_SERIAL"), os.Getenv("TF_ACC_MERAKI_MX_LICENCE")),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("OrganizationsClaim.test", "id", "example-id"),
+
+					resource.TestCheckResourceAttr("organizations_claim.test", "orders.#", "1"),
+					resource.TestCheckResourceAttr("organizations_claim.test", "orders.0", os.Getenv("TF_ACC_MERAKI_ORDER_NUMBER")),
+
+					resource.TestCheckResourceAttr("organizations_claim.test", "serials.#", "1"),
+					resource.TestCheckResourceAttr("organizations_claim.test", "serials.0", os.Getenv("TF_ACC_MERAKI_MX_SERIAL")),
+
+					resource.TestCheckResourceAttr("organizations_claim.test", "licenses.#", "1"),
+					resource.TestCheckResourceAttr("organizations_claim.test", "licenses.0.key", os.Getenv("TF_ACC_MERAKI_MX_LICENCE")),
+					resource.TestCheckResourceAttr("organizations_claim.test", "licenses.0.mode", "addDevices"),
+				),
+			},
+		},
+
+		})
+}
+
+// testAccOrganizationsClaimResourceConfigCreateOrganization is a constant string that defines the configuration for creating an organization resource in your tests.
+const testAccOrganizationsClaimResourceConfigCreateOrganization = `
+ resource "meraki_organization" "test" {
+ 	name = "test_meraki_organizations_claim"
+ 	api_enabled = true
+ }
+ `
+
+// testAccOrganizationsClaimResourceConfigCreate is a constant string that defines the configuration for creating and reading a organizations_claim resource in your tests.
+// It depends on both the organization and network resources.
+func testAccOrganizationsClaimResourceConfigCreate(order, serial, licence string) string {
```

Comment:
Once a license, serial, or order number is claimed there is no way to manually remove it so this is a test which can only be ran a single time. 

Possibly you can move a lic or serial between orgs (DO_NOT_DELETE) however that is a separate resource and out of scope here so we will need to think about ways to better run this test or comment it out completely.

Reply:
So it turns out that it is possible to unclaim a list of serials at the org level: {{baseUrl}}/organizations/:organizationId/inventory/release

I recommend adding this as part of a for loop to remove any serials in the list on the DELETE call.

Reply:
added this {{baseUrl}}/organizations/:organizationId/inventory/release api call in delete call